resources/views/platform/components/developer/inspector-toolbox.blade.php

```diff
@@ -0,0 +1,167 @@
+                // Creates an array of checked achievement IDs and sends it to the updateAchievements function
+                const checkboxes = document.querySelectorAll('[name^=\'achievement\']');
+                const achievements = [];
+                for (let i = 0, n = checkboxes.length; i < n; i += 1) {
+                    if (checkboxes[i].checked) {
+                        achievements.push(checkboxes[i].getAttribute('value'));
+                    }
+                }
+
+                if (!achievements.length || !confirm(getConfirmMessage(property, newValue, achievements.length))) {
+                    return;
+                }
+
+                window.showStatusMessage('Updating...');
+
+                const requestUrl = property === 'flag'
+                    ? '/request/achievement/update-flag.php'
+                    : '/request/achievement/update-type.php';
+
+                $.post(requestUrl, {
+                    achievements,
+                    [property]: newValue
+                }).done(function () {
+                    location.reload();
+                });
+            },
+        }
+    }
+    </script>
+
+    <h3>Toolbox</h3>
+
+    <div
+        class="flex flex-col gap-y-1 [&>button]:flex [&>a]:flex [&>button]:justify-center [&>a]:justify-center [&>button]:py-2 [&>a]:py-2 [&>a]:w-full"
+        x-data="toolboxComponent()"
+    >
+        <button class="btn" @click="refreshPage">Refresh Page</button>
+
+        @if ($modificationLevel === 'full')
+            <button
+                class="btn"
+                @click="updateAchievementsProperty('flag', {{ $isManagingCoreAchievements ? $unofficialFlag : $officialFlag }})"
+            >
+                @if ($isManagingCoreAchievements)
+                    Demote Selected
+                @else
+                    Promote Selected
+                @endif
+            </button>
+        @endif
+
+        <a
+            class="btn"
+            href="/achievementinspector.php?g={{ $gameId }}{{ $isManagingCoreAchievements ? '&f=' . $unofficialFlag : '' }}"
+        >
+            @if ($isManagingCoreAchievements)
+                Unofficial Achievement Inspector
+            @else
+                Core Achievement Inspector
+            @endif
+        </a>
+
+        @if ($canHaveTypes)
+            <button
+                class="btn"
+                @click="updateAchievementsProperty('type', '{{ $progressionType }}')"
+            >
+                Set Selected to Progression
+            </button>
+
+            <button
+                class="btn"
+                @click="updateAchievementsProperty('type', '{{ $winConditionType }}')"
+            >
+                Set Selected to Win Condition
+            </button>
+        @endif
+
+        <button
+            class="btn"
+            @click="updateAchievementsProperty('type', null)"
+        >
+            Set Selected to No Type
+        </button>
```

Comment:
I assume the "Set Selected to No Type" was intentionally not tied to the `$canHaveTypes`? 

Similarly, the "Type: none" is still present in the individual rows.

Reply:
I did leave it intentionally enabled/visible, but on second thought it doesn't make much sense to do so.

After we ship, we can run this query: https://discord.com/channels/476211979464343552/1174126386349342901 and we shouldn't have the problem anymore. I've hidden this button on latest.